### DISCUSSION TITLE
Server actions with next-intl don't seem to be working (404 Not found)

### DISCUSSION THREAD

**Discussion by zkSoju** [vercel/community#7196] [Bug Report] [answered]
### Page to Investigate

https://github.com/0xHoneyJar/minimal-reprod

### Steps to Reproduce

This seems to only happen in production when deployed to vercel. It says the route is not found. Here is a minimal repo of the bug.

<img width="1086" alt="Screenshot 2024-06-13 at 2 14 19 AM" src="https://github.com/vercel/community/assets/28630104/da49ee3a-bbe5-44b7-afcc-5919c19ddd12">

**Comment by swarnava** [comment]
Hi,

Can you share the deployment ID so we can take a closer look? You can locate under `vercel.com/team-slug/project-slug/deploymentid`

**Reply by zkSoju** [reply]
Here you go sir `3AVK62MGtvwL1BaqKutUmXDux2Fi`

**Reply by swarnava** [reply, marked as answer]
We rolled back a change in your account. Can you try deploying again and let us know how it goes?

**Reply by swarnava** [reply]
@zkSoju Can you confirm if you are still facing issue?

**Reply by zkSoju** [reply]
> We rolled back a change in your account. Can you try deploying again and let us know how it goes?

Yessir. Seems to have fixed the issue, could I ask what changed?

**Reply by swarnava** [reply]
We are currently enhancing the behavior of Server Actions within our infrastructure, which has unfortunately introduced a regression. We have temporarily rolled back the changes as we work to identify the root cause and implement a fix. We apologize for any inconvenience this may have caused.

Additionally, if you need further assistance or wish to create a support ticket, please visit https://vercel.com/help.

Thank you for your understanding and patience.

[a comment by an automated account is not shown]
